app_dart/lib/src/request_handlers/refresh_github_commits.dart

```diff
@@ -61,8 +61,21 @@ class RefreshGithubCommits extends ApiRequestHandler<Body> {
         config, branchHttpClientProvider, log, gitHubBackoffCalculator);
 
     for (Branch branch in branches) {
+      int hours = config.existingBranchHours;
+
+      if (branch.name != 'master') {
+        final Set<Commit> commits = await datastore
+            .queryRecentCommits(limit: 1, branch: branch.name)
+            .toSet();
+
+        /// Set [hours] to be longer if a new branch is found.
```

Comment:
Is this to trigger tests when the branch is created? if that is the case is there a way to get the commit the branch was created on and using it as the start point rather than going back several days in the query.

Reply:
From github API https://developer.github.com/v3/repos/commits/#list-commits-on-a-repository, `since` seems the only parameter we can use.

Reply:
Not a blocker for this PR but we can optimize this logic by selecting the last processed commit from the datastore, getting its datetime and grabbing only the commits after that date time. Can you please open a bug for that and add a TODO here?

Reply:
Sounds good! Created: https://github.com/flutter/flutter/issues/54284